### PR TITLE
Fix for #20192 Add settings for min number of Axis

### PR DESCRIPTION
**Pull request by vtkalek**

### CODE REVIEW

File: capabilities.json [hunk 1]
```diff
@@ -188,7 +188,7 @@
             "properties": {
                 "minValue": {
                   "displayNameKey": "Visual_AxisStart",
-                  "displayName": "Axis start",
+                  "displayName": "Axis shift",
```
Comment (ignatvilesov) [inline capabilities.json]: Why did you rename this?

File: src/radarChart.ts [hunk 2]
```diff
@@ -1142,7 +1142,7 @@ module powerbi.extensibility.visual {
                 {});
         }
 
-        private static parseSettings(dataView: DataView, colorPalette: IColorPalette): RadarChartSettings {
+        public static parseSettings(dataView: DataView, colorPalette: IColorPalette): RadarChartSettings {
```
Comment (ignatvilesov) [inline src/radarChart.ts]: Are you going to increase version of the visual?

Comment (vtkalek) [inline src/radarChart.ts]: no because it was not published since last PR

File: capabilities.json (hunk 1 above)
Comment (vtkalek) [inline capabilities.json]: Yes we discussed with Nikita and Ilfat

File: src/radarChart.ts [hunk 3]
```diff
@@ -1155,15 +1155,32 @@ module powerbi.extensibility.visual {
...
-                    }
+
+                    RadarChart.countMinValueForDisplaySettings(minValue, settings);
                 }
             }
             return settings;
         }
 
-        private enumerateDataPoint(): VisualObjectInstance[] {
+        public static countMinValueForDisplaySettings(minValue: any, settings: RadarChartSettings) {
```
Comment (ignatvilesov) [inline src/radarChart.ts]: Please increase version of the visual

Comment (vtkalek) [inline src/radarChart.ts]: done - updated version

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,6 @@
+## 1.0.5
+* Updated asis settings "Asis shift" - added limitations
```
Comment (ignatvilesov) [inline CHANGELOG.md]: `Axis shift`
